src/components/Sidebar/SidebarMenuItem/SidebarMenuItem.spec.tsx

```
@@ -0,0 +1,129 @@
+  it('should apply correct styles for admin variant', () => {
+    renderWithRouter(<SidebarMenuItem {...defaultProps} variant="admin" />);
+
+    const button = screen.getByTestId('dashboardBtn');
+    // Button should have either sidebarBtn or sidebarBtnActive class
+    const hasCorrectClass = button.className.includes('sidebarBtn');
+    expect(hasCorrectClass).toBe(true);
+  });
+
+  it('should handle SVG icons correctly in admin variant', () => {
+    const SvgIcon = () => (
+      <svg data-testid="svg-icon">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)